libs/klv/src/main/java/org/codice/alliance/libs/klv/AttributeNameConstants.java

```diff
@@ -22,17 +28,19 @@
  */
 public class AttributeNameConstants {
 
-    public static final String TEMPORAL_START = "temporal.start";
+    public static final String TEMPORAL_START = DateTime.START;
```

Comment:
Originally, this file was my central repository of attribute names before things like DateTime.END existed, and I've decided to keep it that way so there is one central location that lists all attribute names being used within video processing.